### PR TITLE
`<MultiSelectCheckbox/>` - add missing documentation about testkit methods arguments and missing types

### PR DESCRIPTION
**Pull request by mykas**
This pr is follow up pr after fixing getLabel testkit methods.

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6513

**Review by Zivilekin — APPROVED** [review]

File: src/MultiSelectCheckbox/MultiSelectCheckbox.driver.d.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export interface MultiSelectCheckboxDriver
   extends OmitPolyfill<InputWithOptionsDriver, 'driver'> {
   driver: {
     getNumOfLabels: () => number;
-    getLabels: () => string[];
-    getLabelAt: (index: number) => string;
+    getLabels: (delimiter:string) => string[];
+    getLabelAt: (index: number, delimiter:string) => string;
```
Comment (splintor) [inline src/MultiSelectCheckbox/MultiSelectCheckbox.driver.d.ts:9]: Shouldn't the `delimiter` parameter be optional?
```
    getLabels: (delimiter = ',') => string[];
    getLabelAt: (index: number, delimiter = ',') => string;
```

File: src/MultiSelectCheckbox/MultiSelectCheckbox.uni.driver.d.ts [hunk 2]
```diff
@@ -6,8 +6,8 @@ export interface MultiSelectCheckboxUniDriver
   extends OmitPolyfill<InputWithOptionsUniDriver, 'driver'> {
   driver: {
     getNumOfLabels: () => Promise<number>;
-    getLabels: () => Promise<string[]>;
-    getLabelAt: (index: number) => Promise<string>;
+    getLabels: (delimiter: string) => Promise<string[]>;
+    getLabelAt: (index: number, delimiter: string) => Promise<string>;
```
Comment (splintor) [inline src/MultiSelectCheckbox/MultiSelectCheckbox.uni.driver.d.ts:10]: Shouldn't the `delimiter` parameter be optional?
```
    getLabels: (delimiter = ',') => Promise<string[]>;
    getLabelAt: (index: number, delimiter = ',') => Promise<string>;
```

File: src/MultiSelectCheckbox/README.TESTKIT.md [hunk 3]
```diff
@@ -12,12 +12,12 @@ The `<MultiSelectCheckbox/>` TestKit is exposing the following drivers:
...
-| method | arguments | returned value | description |
-|--------|-----------|----------------|-------------|
-| getNumOfLabels | - | number | returns the number of displayed labels in the readonly input field |
-| getLabels | - | array | returns an array of the labels that displayed in the readonly input field |
-| getLabelAt | number | string | returns a label from the displayed labels - by index |
-| setProps | json | element | returns a clone of this element with the new props from the json |
+| method | returned value | description |
+|--------|----------------|-------------|
+| getNumOfLabels | number | returns the number of displayed labels in the readonly input field |
+| getLabels(delimiter:string) | array | returns an array of the labels that displayed in the readonly input field |
```
Comment (splintor) [inline src/MultiSelectCheckbox/README.TESTKIT.md:18]: Shouldn't the `delimiter` parameter be optional?
```
| getLabels(delimiter?: string) | array | returns an array of the labels that displayed in the readonly input field |
| getLabelAt(index: number, delimiter?: string) | string | returns a label from the displayed labels - by index |
```

**Review by splintor — COMMENTED** [review]
The types and docs should indicate that the `delimiter` param is optional and has a default value of `','`.

**Pull request merged**